docs(README): Logo, move AUR to download section, new badge

ThatOneCalculator opened the pull request:
:)

![image](https://github.com/user-attachments/assets/454b404d-b514-4fc4-b8f3-11676b0fee2a)

ThatOneCalculator committed:
docs: move AUR to download section, new badge

ThatOneCalculator committed:
docs: add arch logo to AUR badge

ThatOneCalculator committed:
docs: logo

ADBeveridge commented:
Hey @ThatOneCalculator thanks for the pr, it made me realize that I needed to update the badge situation (like build status etc). So if you don't mind I'll commit some changes on your branch then merge the pr. Also thanks for taking up the aur package, appreciated :)

ThatOneCalculator commented:
Go for it! Ty!

ADBeveridge committed:
Update badges

Make AUR and COPR have similar download badges, and made the official badges at the top only for Flathub.
